example_analysis/merge_3/README.md

```diff
@@ -0,0 +1,83 @@
+# merge_3
+
+This module provides a workflow for aligning and merging phenotype microscopy data with sequencing-by-synthesis (SBS) data. It uses triangle hashing algorithms for robust point matching between different imaging modalities.
+
+## Contents
+
+1. `merge_3_smk_test.ipynb`: Jupyter notebook for testing alignment parameters and visualizing results
+2. `merge_3.smk`: Main Snakemake file for merging aligned phenotype and SBS data
+3. `hash_3.smk`: Snakemake file for generating hash alignments
+4. `merge_3.sh`: Bash script to execute the complete workflow
+5. `merge_3_eval.py`: Python script for collating alignment and phenotype data, filtering, and generating quality control plots.
```

Comment:
That's a good point. I think for now, these steps definitely work in unison, but because of snakemake rules, we can't have a single snakemake that runs both. I spent a few days trying to get around this but it didn't materially change anything, so I gave up.

We should move to align in brieflow. Good point.